app/src/sharedTest/java/org/oppia/android/app/settings/profile/ProfileResetPinActivityTest.kt

```diff
@@ -621,7 +621,7 @@ class ProfileResetPinActivityTest {
   }
 
   @Test
-  fun testResetPin_asUser_inputWrongConfirmPin_clickSave_checkConfirmWrongError() {
+  fun testProfileResetPin_withUser_inputWrongConfirmPin_save_checkConfirmWrongError() {
```

Comment:
testProfileResetPin_withUser_inputWrongConfirmPin_save_confirmWrongErrorIsDisplayed

Reply:
Exceeded the limit, instead changed confirmWrongErrorIsDisplayed to confirmWrongErrorIsShown throughout the file, similarly for other functions.

Reply:
testProfileResetPin_withUser_inputWrongConfirmPin_save_confirmWrongErrorIsDisplayed

It's not exceeding the 100 char length

Reply:
Sorry, I meant it in another comment @ line 432. (shifted to https://github.com/oppia/oppia-android/pull/2433#discussion_r554508025)